Userland/Utilities/unzip.cpp

```diff
@@ -140,6 +143,28 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         TRY(Core::System::chdir(output_directory_path));
     }
 
+    if (list_files) {
+        outln("  Length     Date      Time     Name");
+        outln("--------- ---------- --------   ----");
+        u32 members_count = 0;
+        u64 total_size = 0;
+        TRY(zip_file->for_each_member([&](auto zip_member) -> ErrorOr<IterationDecision> {
+            members_count++;
+
+            auto time = time_from_packed_dos(zip_member.modification_date, zip_member.modification_time);
+            auto time_str = TRY(Core::DateTime::from_timestamp(time.to_seconds()).to_string());
```

Comment:
This code isn't building. Looks like some of the APIs changed since your originally submitted this. I think this is enough to fix it:

```suggestion
            auto time_str = TRY(Core::DateTime::from_timestamp(time.seconds_since_epoch()).to_string());
```

In future, if you can work on a separate branch instead of on `master`, it's easier to then rebase your changes to include anything that others have changed. :^)